### PR TITLE
add libdrm-amdgpu-dev for focal image with empty hipDeviceGetName()

### PR DESCRIPTION
**Pull request by i-chaochen**
https://github.com/ROCm/frameworks-internal/issues/6949

This patch also needs go to 
- [x]  develop-upstream-QA-rocm61 https://github.com/ROCm/tensorflow-upstream/pull/2392
- [x]  r2.15-rocm-enhanced https://github.com/ROCm/tensorflow-upstream/pull/2393
- [x] r2.14-rocm-enhanced https://github.com/ROCm/tensorflow-upstream/pull/2394
- [x] r2.13-rocm-enhanced https://github.com/ROCm/tensorflow-upstream/pull/2395

### CODE REVIEW

**Comment by i-chaochen** [comment]
retest Ubuntu-CPU please

File: tensorflow/tools/ci_build/Dockerfile.rocm [hunk 1]
```diff
@@ -20,12 +20,12 @@ RUN apt-get clean all
 RUN wget -qO - https://repo.radeon.com/rocm/rocm.gpg.key | apt-key add -;
 RUN wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -;
 RUN bin/bash -c 'if [[ $ROCM_DEB_REPO == https://repo.radeon.com/rocm/*  ]] ; then \
-      echo "deb [arch=amd64] $ROCM_DEB_REPO $ROCM_BUILD_NAME $ROCM_BUILD_NUM" > /etc/apt/sources.list.d/rocm.list; \
+      echo "deb [arch=amd64] $ROCM_DEB_REPO $ROCM_BUILD_NAME $ROCM_BUILD_NUM" > /etc/apt/sources.list.d/rocm.list; \
```
Comment (jayfurmanek) [inline tensorflow/tools/ci_build/Dockerfile.rocm]: whitespace lint on this line

**Comment by jayfurmanek** [comment]
You need to add the repo here as well:
https://github.com/ROCm/tensorflow-upstream/blob/develop-upstream/tensorflow/tools/tf_sig_build_dockerfiles/setup.rocm.sh

**Comment by i-chaochen** [comment]
@jayfurmanek the change is done. PTAL

**Review by jayfurmanek — APPROVED** [review]

**Comment by i-chaochen** [comment]
retest Ubuntu-CPU please

**Comment by i-chaochen** [comment]
retest Ubuntu-CPU please

**Pull request merged**